scripts/bootstrap_env.sh

```
@@ -93,9 +93,9 @@ load_from_active_fetcher() {
 
         # Merge with output
         if [[ -f "$temp_output" ]]; then
+            local var_count=$(grep -c '^[A-Z_]=' "$temp_output" 2>/dev/null || echo "0")
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)